src/main/java/com/esri/samples/analysis/distance_measurement_analysis/DistanceMeasurementAnalysisController.java

```diff
@@ -0,0 +1,169 @@
+import javafx.fxml.FXML;
+import javafx.geometry.Point2D;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+
+import com.esri.arcgisruntime.UnitSystem;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geoanalysis.LocationDistanceMeasurement;
+import com.esri.arcgisruntime.geometry.Distance;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.AnalysisOverlay;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+public class DistanceMeasurementAnalysisController {
+
+  @FXML private SceneView sceneView;
+  @FXML private Label directDistanceLabel;
+  @FXML private Label verticalDistanceLabel;
+  @FXML private Label horizontalDistanceLabel;
+  @FXML private ComboBox<UnitSystem> unitSystemComboBox;
+
+  private LocationDistanceMeasurement distanceMeasurement;
+
+  public void initialize() {
+
+    // create a scene and set it to the scene view
+    ArcGISScene scene = new ArcGISScene();
+    scene.setBasemap(Basemap.createImagery());
+    sceneView.setArcGISScene(scene);
+
+    // add base surface for elevation data
+    Surface surface = new Surface();
+    surface.getElevationSources().add(new ArcGISTiledElevationSource("http://elevation3d.arcgis" +
+        ".com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"));
+    surface.getElevationSources().add(new ArcGISTiledElevationSource("https://tiles.arcgis.com/tiles/d3voDfTFbHOCRwVR/arcgis/rest/services/MNT_IDF/ImageServer"));
+    scene.setBaseSurface(surface);
+
+    final String buildings = "http://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_Brest/SceneServer/layers/0";
+    ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer(buildings);
+    scene.getOperationalLayers().add(sceneLayer);
+
+    // create an analysis overlay and add it to the scene view
+    AnalysisOverlay analysisOverlay = new AnalysisOverlay();
+    sceneView.getAnalysisOverlays().add(analysisOverlay);
+
+    // initialize a distance measurement and add it to the analysis overlay
+    Point start = new Point(-4.494677, 48.384472, 24.772694, SpatialReferences.getWgs84());
+    Point end = new Point(-4.495646, 48.384377, 58.501115, SpatialReferences.getWgs84());
+    distanceMeasurement = new LocationDistanceMeasurement(start, end);
+    analysisOverlay.getAnalyses().add(distanceMeasurement);
+
+    // zoom to the initial measurement
+    sceneView.setViewpointCameraAsync(new Camera(start, 200.0, 0.0, 45.0, 0.0));
```

Comment:
i would just set this to `setViewpointCamera` so that when the application starts it already at the location and doesn't do that slow zoom in.